src/protein/ngl-util.js

```diff
@@ -65,13 +70,13 @@ export function init(parentId, childId, aligned, active, query_display_mode, mot
 
   Promise.all([
     stage.loadFile(`rcsb://${parentId}`).then((o) => {
-      o.addRepresentation(query_display_mode, { sele: select1, color: '#2AF598', surfaceType: query_surface_rep });
+      o.addRepresentation(query_display_mode, { sele: select1, color: '#2AF598', surfaceType: query_surface_rep});
       o.autoView();
       return o;
     }),
 
     stage.loadFile(`rcsb://${childId}`).then((o) => {
-      o.addRepresentation(motif_display_mode, { sele: select2, color: '#20BDFF', surfaceType: motif_surface_rep });
+      o.addRepresentation(motif_display_mode, { sele: select2, color: '#20BDFF', surfaceType: motif_surface_rep});
```

Comment:
These have leading spaces in the brackets, why do need to remove the trailing ones?
Here, and line 73